partiql-tests-data/eval/primitives/path.ion

```diff
@@ -353,412 +433,480 @@ path::[
     }
   },
   {
-    name:"pathWildCardOverScalar",
-    statement:"s[*]",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        "hello"
-      ]
-    }
+    name: "pathWildCardOverScalar",
+    statement: "s[*]",
+    assert: [
+      {
+        evalMode: EvalModeCoerce,
+        result: EvaluationSuccess,
+        output: $bag::[
+          "hello"
+        ]
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail
+      }
+    ]
   },
   {
-    name:"pathUnpivotWildCardOverScalar",
-    statement:"s.*",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        "hello"
-      ]
-    }
+    name: "pathUnpivotWildCardOverScalar",
+    statement: "s.*",
+    assert: [
+      {
+        evalMode: EvalModeCoerce,
+        result: EvaluationSuccess,
+        output: $bag::[
+          "hello"
+        ]
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail
```

Comment:
Could you please elaborate on why this failed?

```
SELECT VALUE v1 
FROM UNPIVOT s as v1
```

Reply:
Spec section 5.1.1 - Iteration over a scalar value

FROM s AS V
where s is a scalar value. Then s coerces into the bag << s >>. 

This section seems to be mode independent, i.e., coercion always happens.

Reply:
At the top of Section 5.1.1:
> In the following cases the expression in the FROM clause item has the wrong type. **Under the type checking option, all of these cases raise an error and the query fails. Under the permissive option, the cases proceed as follows**...